entrypoint.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+
+export PATH=/usr/bin:/bin:/usr/sbin:/sbin
```

Comment:
You specify a default PATH here that doesn't include the path where you install the server application, i.e. /usr/local/bin right now.

In the long run we may have other tools being installed to that path, and it is possible that the server, or just the entry point script itself, may need to run them, so it may be a good idea to explicitly add the bin directory where the server and related tools are installed to the script's default PATH settings.

Note that if we install under /usr, rather than under /usr/local, no change is needed here.

Reply:
Updated to install the executable under /usr/bin